rtf

```diff
@@ -321,6 +327,49 @@ class Platform:
     print("export RUDDER_TOKEN=" + os.environ['RUDDER_TOKEN'])
     print("alias rcli='./rudder-cli --skip-verify --url=" + rudder_url + " --token=" + token + "'")
 
+  def export_test(self, rule_uuid, test_name, scenario=False):
+    """ Export a given rule, for use in a test or a scenario """
+    # Retrieve data
+    (rudder_url, token) = self.get_server_info()
+    endpoint = RudderEndPoint(rudder_url, token, verify=False)
+    rule = endpoint.rule_details(rule_uuid)['rules'][0]
+    directives = []
+    for directive_uuid in rule['directives']:
+      directive = endpoint.directive_details(directive_uuid)['directives'][0]
+      del directive['id']
+      directives.append(directive)
+
+    # create test files
+    rule_file = "spec/tests/"+test_name+"_rule.rb"
+    test_file = "spec/tests/"+test_name+"_test.rb"
+    scenario_file = "scenario/"+test_name+".py"
+
+    make_rule_testfile(rule_file, rule, directives)
+    make_user_testfile(test_file)
+    if scenario:
```

Comment:
Or at least:

    print("""
      the big thing
    """) % { vars }
    if scenario:
      make_scenario_file(scenario_file, test_name)
      print("Additionnaly, a scenario has been created in " + scenario_file)